<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_LoadImage">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LoadImage">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_VerticalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Thresholding</string>
    </property>
    <widget class="QRadioButton" name="radioButton_Threshold_Binary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Binary</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Threshold_Binary_Inv">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Binary_Inv</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Prim_Plan">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Markers">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Watershed">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>80</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_PrimPlan">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Prim plan</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Marker">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Markers</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Watersheds">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Watersheds</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
